src/filter/mod.rs

```diff
@@ -276,9 +156,11 @@ impl<'a, K: Num + Copy + 'a> Kernel<'a, K> {
     }
 }
 
+/// The gaussian probability density function with a mean of zero.
 #[inline]
-fn gaussian(x: f32, r: f32) -> f32 {
-    ((2.0 * f32::consts::PI).sqrt() * r).recip() * (-x.powi(2) / (2.0 * r.powi(2))).exp()
+fn gaussian_pdf(x: f32, sigma: f32) -> f32 {
+    // Equation derived from <https://en.wikipedia.org/wiki/Gaussian_function>
+    (sigma * f32::consts::TAU.sqrt()).recip() * (-x.powi(2) / (2.0 * sigma.powi(2))).exp()
```

Comment:
Let's leave this as `PI`. `TAU` is far more obscure.

Reply:
My thoughts were purely on code performance and not readability, but the compiler probably optimizes it anyway so I'll revert to `2*pi`.

Reply:
👍 This should be the easiest possible case for constant folding so I expect both versions to produce identical code.